curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -2243,17 +2262,19 @@ CURVEFS_ERROR DataCache::Flush(uint64_t inodeId, bool toS3) {
                 }
                 VLOG(9) << "PutObjectAsyncCallBack: " << context->key
                         << " pendingReq is: " << pendingReq;
+                if (cachePoily::RCache == cachePoily) {
```

Comment:
Only ReadCache，WRCache will do `s3ClientAdaptor_->GetDiskCacheManager()->Enqueue(*iter);`，in which it will wirte to WCache before uploading then doing upload async.

Reply:
> Only ReadCache，WRCache will do `s3ClientAdaptor_->GetDiskCacheManager()->Enqueue(*iter);`，in which it will wirte to WCache before uploading then doing upload async.

ok